tests/integration/config.go

```diff
@@ -131,12 +168,11 @@ func DefaultTestRun() TestRun {
 	}
 }
 
-func (s *TestRun) ParseCLIFlags() {
-	localSdkPath := flag.String("local-sdk-path", "",
-		"path of a local sdk version to build and reference in integration tests")
-	flag.Parse()
-	s.localSdkPath = *localSdkPath
-	fmt.Println(s.localSdkPath)
+func (s *TestRun) SetLocalSDKPath(path string) {
```

Comment:
Still not working for me locally, maybe we just keep the old method which we know works?

Reply:
Check now. There was a race condition in `start-docker.sh` when running in parallel.

The script is run once for each container, so while the first container was using the dir (copying into it), another container would attempt to delete the folder, causing a race condition.